HTML/Vehicles.html

```diff
@@ -1,8 +1,128 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-</head>
-<body bgcolor = "violet">
-<h1> This is the Vehicles page </h1>
+    <style>

```

Comment:
Would want this on an external css files. Especially since we have a folder dedicated to it

Reply:
yeah I'll do that, sorry I didn't notice the folder

Reply:
No worries!